bindings/azure/storagequeues/storagequeues_test.go

```diff
@@ -221,7 +222,7 @@ func TestReadQueueDecode(t *testing.T) {
 
 	time.Sleep(30 * time.Second)
 
-}
+}.
```

Comment:
Really? a dot here? Is there a way to skip lint for this case?

Reply:
Fixed.